frontend/src/styles/GlobalStyle.ts

```diff
@@ -86,6 +86,23 @@ export const globalStyle = kialiStyle({
           color: 'var(--pf-v5-global--link--Color--light--hover)'
         }
       }
+    },
+
+    /**
+     * Hide kebab menu from Patternfly topology groups
+     */
+    '& .pf-topology__group__label': {
```

Comment:
PFT groups don't have a `labelClassName` property, so I have to modify the group label CSS in the global style.